lib/charon/utils/key_generator.ex

```diff
@@ -42,7 +42,15 @@ defmodule Charon.Utils.KeyGenerator do
       iterations = opts[:iterations] || 250_000
 
       :crypto.pbkdf2_hmac(digest, base_secret, salt, iterations, length)
-      |> tap(&FastGlobal.put(__MODULE__, Map.put(cache, {base_secret, salt, opts}, &1)))
+      |> tap(&:persistent_term.put(__MODULE__, Map.put(cache, {base_secret, salt, opts}, &1)))
+    end
+  end
+
+  defp get_cache() do
+    try do
+      :persistent_term.get(__MODULE__)
+    rescue
+      _ -> %{}
     end
```

Comment:
Is deze functie niet gelijk aan `:persistent_term.get(__MODULE__, %{})`?

https://www.erlang.org/doc/apps/erts/persistent_term.html#get/2

Reply:
I must concur with my colleague Dani on this point—the use of `try rescue` may indeed be superfluous. Aside from that, an excellent suggestion!

Reply:
Missed the /2 variant, of course it is equivalent :)